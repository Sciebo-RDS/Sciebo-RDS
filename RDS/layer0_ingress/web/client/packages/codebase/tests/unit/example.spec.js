import { createLocalVue, mount } from '@vue/test-utils'
import HelloWorld from '@/components/HelloWorld.vue'
import Vuetify from 'vuetify'

describe('HelloWorld.vue', () => {
  let localVue
  let vuetify

  beforeEach(() => {
    localVue = createLocalVue()
    vuetify = new Vuetify()
  })

  it('should work', () => {
    const wrapper = mount(HelloWorld, {
      localVue,
      vuetify,
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})

