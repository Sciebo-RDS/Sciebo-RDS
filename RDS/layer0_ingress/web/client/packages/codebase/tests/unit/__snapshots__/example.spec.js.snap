// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue should work 1`] = `
<div class="container">
  <div class="row text-center">
    <div class="col col-12">
      <div class="v-image v-responsive my-3 theme--light" style="height: 200px;">
        <div class="v-responsive__content"></div>
      </div>
    </div>
    <div class="mb-4 col">
      <h1 class="display-2 font-weight-bold mb-3">
        Welcome to Vuetify
      </h1>
      <p class="subheading font-weight-regular">
        For help and collaboration with other Vuetify developers,
        <br>please join our online
        <a href="https://community.vuetifyjs.com" target="_blank">Discord Community</a>
      </p>
    </div>
    <div class="mb-5 col col-12">
      <h2 class="headline font-weight-bold mb-3">
        What's next?
      </h2>
      <div class="row justify-center"><a href="https://vuetifyjs.com/components/api-explorer" target="_blank" class="subheading mx-3">
          Explore components
        </a><a href="https://vuetifyjs.com/getting-started/pre-made-layouts" target="_blank" class="subheading mx-3">
          Select a layout
        </a><a href="https://vuetifyjs.com/getting-started/frequently-asked-questions" target="_blank" class="subheading mx-3">
          Frequently Asked Questions
        </a></div>
    </div>
    <div class="mb-5 col col-12">
      <h2 class="headline font-weight-bold mb-3">
        Important Links
      </h2>
      <div class="row justify-center"><a href="https://vuetifyjs.com" target="_blank" class="subheading mx-3">
          Documentation
        </a><a href="https://community.vuetifyjs.com" target="_blank" class="subheading mx-3">
          Chat
        </a><a href="https://madewithvuejs.com/vuetify" target="_blank" class="subheading mx-3">
          Made with Vuetify
        </a><a href="https://twitter.com/vuetifyjs" target="_blank" class="subheading mx-3">
          Twitter
        </a><a href="https://medium.com/vuetify" target="_blank" class="subheading mx-3">
          Articles
        </a></div>
    </div>
    <div class="mb-5 col col-12">
      <h2 class="headline font-weight-bold mb-3">
        Ecosystem
      </h2>
      <div class="row justify-center"><a href="https://github.com/vuetifyjs/vuetify-loader" target="_blank" class="subheading mx-3">
          vuetify-loader
        </a><a href="https://github.com/vuetifyjs/vuetify" target="_blank" class="subheading mx-3">
          github
        </a><a href="https://github.com/vuetifyjs/awesome-vuetify" target="_blank" class="subheading mx-3">
          awesome-vuetify
        </a></div>
    </div>
  </div>
</div>
`;
